### DISCUSSION TITLE
Problem with Classifier and Window Area

### DISCUSSION THREAD

**Discussion by hanan-commits-work** [iTwin/itwinjs-core#3235] [Q&A] [answered]
Greetings.
When I add the classifier: [ ] as a prop in ContextRealityProps, it somehow causes a bug in the Window Area tool. Basically my reality model is filled with bunch of classifiers, it worked fine in 2.1x but in 3.0 the similar logic causes window area tool to be buggy. The bug is specifically that, "Too close to Target". Even though it is far away.
I am following these samples

https://www.itwinjs.org/sample-showcase/?group=Viewer+Features&sample=classifier-sample&context=Metrostation+Sample&imodel=Philadelphia

https://www.itwinjs.org/sample-showcase/?group=Viewer&sample=reality-data-sample

**Comment by pmconne** [comment]
Supply your code.

**Comment by hanan-commits-work** [comment, marked as answer]
Issue is resolved. There was problem with Margin Options Argument. I didnt place margin options in ZoomToRange thats why it was giving this problem
